packages/devextreme/js/__internal/scheduler/tooltip_strategies/m_mobile_tooltip_strategy.ts

```diff
@@ -65,13 +65,27 @@ export class MobileTooltipStrategy extends TooltipStrategyBase {
     return false;
   }
 
-  _onShowing() {
+  private setTooltipConfig(): void {
     const isTabletWidth = getWidth(getWindow()) > 700;
 
-    this._tooltip.option('height', MAX_HEIGHT.DEFAULT);
-    const listHeight = getOuterHeight(this._list.$element());
+    const listHeight = getOuterHeight(this._list.$element().find('.dx-scrollable-content'));
```

Comment:
It's better to create the separate ```const``` variable for CSS selectors:
```suggestion
    const listHeight = getOuterHeight(this._list.$element().find(CLASS.scrollableContent));
```